mine_text2.py

```diff
@@ -0,0 +1,64 @@
+from nltk import word_tokenize
+from nltk.corpus import stopwords
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+from collections import Counter
+import numpy as np
+import matplotlib.pyplot as plt
+
+# koran.txt has koran excerpt manually copied and pasted from koran.pickle
+with open('koran.txt', 'r') as myfile1:
+    koran = myfile1.read().replace('\n', ' ')
+
+# analyzing sentiment
+analyzer = SentimentIntensityAnalyzer()
+k_sent = analyzer.polarity_scores(koran)
+print(k_sent)
+
+
+def filter_text(text):
+    '''
+    This function removes all characters in a string that are not alpha
+    or spaces. In addition, it handles the wrapping of words around the end
+    of a line with by removing the '- '.
+    '''
+    text = text.lower()
+    text = text.replace('- ', '')
+    text_filtered = ''
+    alpha = 'abcdefghijklmnopqrstuvwxyz '
+    for char in text:
+        if char in alpha:
+            text_filtered += char
+    return text_filtered
+
+
+def important_tokens(text):
+    '''
+    This function goes through text and creates a list of all words in the
+    text that do not appear in the nltk stopwords list.
+    '''
+    stop = stopwords.words('english')
```

Comment:
Most of readers of this code will not know what nltk stopwords are. It would be good to include an in-line comment describing what nltk stopwords are and how they are being used here.